### PR TITLE
display message to test taker when paused due to loss of focus

### PR DESCRIPTION
**Pull request by zagovorichev**
https://oat-sa.atlassian.net/browse/TAO-2732

### CODE REVIEW

File: views/js/controller/runner/runner.js [hunk 1]
```diff
@@ -81,9 +83,23 @@ define([
...
+                    params.reason = data.reason;                    
+                }
+            })
             .after('destroy', function () {
+                
                 // at the end, we are redirected to the exit URL
-                window.location = config.exitUrl;
+                var url = config.exitUrl;
+                if (params && params.reason) {
+                    url = new urlParser(url);
```
Comment (krampstudio) [inline views/js/controller/runner/runner.js]: You don't need the UrlParser but the [util/url](https://github.com/oat-sa/tao-core/blob/master/views/js/util/url.js#L150)

File: views/js/controller/runner/runner.js [hunk 2]
```diff
@@ -73,6 +74,7 @@ define([
             renderTo: $('.runner')
         });
 
+        var params = {};
```
Comment (krampstudio) [inline views/js/controller/runner/runner.js]: Please reflect hoisting in variable déclaration

**Comment by zagovorichev** [comment]
@krampstudio Update

File: views/js/controller/runner/runner.js [hunk 3]
```diff
@@ -73,6 +74,7 @@ define([
             renderTo: $('.runner')
         });
 
+        var reason = '';
```
Comment (krampstudio) [inline views/js/controller/runner/runner.js]: please declare your variable on top of the function to reflect hoisting.

**Pull request merged**